tests/phpunit/tests/media.php

```diff
@@ -6445,6 +6445,56 @@ public function test_wp_img_tag_add_auto_sizes( string $input, string $expected
 		);
 	}
 
+	/**
+	 * @ticket 62305
+	 *
+	 * @dataProvider data_provider_to_test_image_converted_to_other_format_has_correct_filename
```

Comment:
Convention is to change the prefix for data providers to a single test.

```suggestion
	 * @dataProvider data_image_converted_to_other_format_has_correct_filename
```